<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TradeHistoryDialog</class>
 <widget class="QWidget" name="TradeHistoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trade History</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Trade History</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTableWidget" name="versionListWidget">
      <property name="minimumHeight">
       <number>150</number>
      </property>
      <property name="maximumHeight">
       <number>250</number>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>Version</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Recorded At</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Modified By</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Commentary</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="detailWidget">
      <layout class="QVBoxLayout" name="detailLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="changesTab">
          <attribute name="title">
           <string>Changes</string>
          </attribute>
          <layout class="QVBoxLayout" name="changesTabLayout">
           <item>
            <widget class="QTableWidget" name="changesTableWidget">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <column>
              <property name="text">
               <string>Field</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Old Value</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Value</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="fullDetailsTab">
          <attribute name="title">
           <string>Full Details</string>
          </attribute>
          <layout class="QVBoxLayout" name="fullDetailsTabLayout">
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>560</width>
                <height>450</height>
               </rect>
              </property>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="externalIdValueLabel">
                 <property name="text">
                  <string>External ID:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="externalIdValue">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="tradeTypeValueLabel">
                 <property name="text">
                  <string>Trade Type:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="tradeTypeValue">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="lifecycleEventValueLabel">
                 <property name="text">
                  <string>Lifecycle Event:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="lifecycleEventValue">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="nettingSetIdValueLabel">
                 <property name="text">
                  <string>Netting Set:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLabel" name="nettingSetIdValue">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="tradeDateValueLabel">
                 <property name="text">
                  <string>Trade Date:</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLabel" name="tradeDateValue">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="effectiveDateValueLabel">
                 <property name="text">
                  <string>Effective Date:</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QLabel" name="effectiveDateValue">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="terminationDateValueLabel">
                 <property name="text">
                  <string>Termination Date:</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QLabel" name="terminationDateValue">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="7" column="0">
                <widget class="QLabel" name="executionTimestampValueLabel">
                 <property name="text">
                  <string>Execution Timestamp:</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="1">
                <widget class="QLabel" name="executionTimestampValue">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="8" column="0">
                <widget class="QLabel" name="versionNumberLabel">
                 <property name="text">
                  <string>Version:</string>
                 </property>
                </widget>
               </item>
               <item row="8" column="1">
                <widget class="QLabel" name="versionNumberValue">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="9" column="0">
                <widget class="QLabel" name="modifiedByLabel">
                 <property name="text">
                  <string>Modified By:</string>
                 </property>
                </widget>
               </item>
               <item row="9" column="1">
                <widget class="QLabel" name="modifiedByValue">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="10" column="0">
                <widget class="QLabel" name="recordedAtLabel">
                 <property name="text">
                  <string>Recorded At:</string>
                 </property>
                </widget>
               </item>
               <item row="10" column="1">
                <widget class="QLabel" name="recordedAtValue">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="11" column="0">
                <widget class="QLabel" name="changeCommentaryLabel">
                 <property name="text">
                  <string>Change Commentary:</string>
                 </property>
                </widget>
               </item>
               <item row="11" column="1">
                <widget class="QLabel" name="changeCommentaryValue">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
